eZ Platform/Admin UI/Field Type Edit/Requirements.md

```diff
@@ -1,45 +1,35 @@
-# Content editing
+# Content Editing
 
+## Objective
 
-# Objective
+The objective (which may be supported by prototypes or examples), is to identify and decide on the best approach to implement CRUD (create, read, update and delete) operations on Field Types within the user interface(s) and APIs exposed by eZ Platform in eZ Platform v2.x.
 
-"As an eZ Engineer, I want...
+The underlying goals are:
 
-To create a functional, experimental implementation of a refactoring of the Field Type API in eZ Platform supporting full CRUD (create, read, update and delete) for content creation and editing. The purpose of the research is to find a common way to define the user interface using different technologies."
+- Simplify how to create a new Field Type (wether it is for eZ Systems' own needs when creating Field Types in the core platform, or developers customizing the platform by creating custom Field Types)
+- Reduce risk of errors and simplify maintenance by limiting the number of places where UI behavior is defined
+- Increase speed for developers to develop and maintain Field Types with a similar efficiency level as what was offered by eZ Publish Legacy, or better.
 
-Underlying goals are:
+## Firm requirements
 
-* Simplify how to create new custom Field Type
-* Reduce errors by limiting the number of places UI definitions are done
-* Increase developer velocity when developing and maintaining Field Types
+- The solution should enable CRUD operations on fields in the exact same way, in both eZ Platform UI (cf. Hybrid application prototype), and in any website directly powered by eZ Platform Web Framework (as opposed to a decoupled website, simply using eZ Platform APIs)
+- The solution should enable CRUD operations on fields in a remote website / web application (in any programming language) using the REST API, and should minimize the UI logic that will have to be implemented for that (e.g. moving as much as possible of the UI logic in the APIs, such as validation rules, etc., or exposing the Web UI through the API)
```

Comment:
irrelevant, I missed the distinction between between _web vs non-web_ applications